community-operators/knative-serving-operator/knative-serving-operator.v0.5.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,209 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: knative-serving-operator
+      deployments:
+      - name: knative-serving-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: knative-serving-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: knative-serving-operator
+            spec:
+              containers:
+              - command:
+                - knative-serving-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-serving-operator
+                image: quay.io/openshift-knative/knative-serving-operator:v0.5.2
+                args:
+                  - --filename=https://raw.githubusercontent.com/openshift/knative-serving/release-v0.5.2/openshift/release/knative-serving-v0.5.2.yaml
+                imagePullPolicy: Always
+                name: knative-serving-operator
+                resources: {}
+              serviceAccountName: knative-serving-operator
+      permissions:
```

Comment:
This permissions section looks good. If the operator doesn't need additional RBAC please remove the clusterPermissions section.

Reply:
See above comment